### PR TITLE
Quick Start Guide Revamp + Misc Changes

### PR DESCRIPTION
**Pull request by brody192**

### CODE REVIEW

File: src/docs/quick-start.md [hunk 1]
```diff
@@ -2,196 +2,167 @@
...
-1. Creating a project from a template
-2. Developing the project locally
-3. Deploying to the cloud
+    **[Option 2](/quick-start#deploying-your-project---with-the-cli)** - Deploying with the [CLI](/guides/cli).
 
-You will need [Node](https://nodejs.org/en/download/) to run the project locally. And you will need either Node or [Homebrew](https://brew.sh/) on your machine to install the Railway CLI. We will use [Yarn](https://yarnpkg.com/) for our examples locally.
+3. **Deploying a template** - Ideal for deploying pre-configured software with minimal effort.
 
-## Create a Project
+For our project, we'll utilize a basic <a href="https://github.com/railwayapp-templates/nextjs-basic" target="_blank">NextJS app</a>, and for the template deployment, we'll use the <a href="https://railway.app/template/umami-analytics" target="_blank">Umami template</a>.
```
Comment (melissa-hale) [inline src/docs/quick-start.md:17]: This is a bit confusing to me.  I'm wondering what it means by "our project" and what is this NextJS app?  I think simply expanding on what exactly this is walking me through would be good.  like "To accomplish the first option above, this guide will demonstrate the process for deploying a project directly from Github"

I think you should also mention that they'll need to link their Github with Railway maybe?  just imagining, what if I'm a user who is using this as a step-by-step guide and I haven't done that part yet.  What if we even included a step that's like "fork this repository" so they can follow along?

File: src/docs/quick-start.md [hunk 2]
```diff
@@ -2,196 +2,167 @@
...
-Under the list of options in the menu, select "Deploy Template".
-
-### Deploy the Next JS Prisma Template
+<Image src="https://res.cloudinary.com/railway/image/upload/v1723752559/docs/quick-start/new_project_uyqqpx.png"
+alt="screenshot of new project menu with deploy from github selected"
+layout="responsive"
+width={836} height={860} quality={100} />
 
-Railway offers 50+ templates ranging from blogs to self-hosted apps. In the Command Palette, type `NextJS Prisma`.
+*Railway requires a valid GitHub account to be linked. If your Railway account isn't associated with one, you will be prompted to link it.*
```
Comment (melissa-hale) [inline src/docs/quick-start.md:32]: lol keep reading Melissa.  still wonder about the fork step, might be good?  i'll go through the rest before i add comments 😄

**Comment by melissa-hale** [comment]
Shaping up really really nicely.  I think all the feedback I have can be addressed by making it a bit more guided, so you can guarantee the experience is exactly how you're describing it in the guide...  Like instead of saying "we're going to deploy this NextJS project", I would say "let's fork this NextJS project and then deploy it"

Same for the CLI deploy.. "clone this repo and then use railway up to deploy it"

Same for the template deploy.. "search for Umami, click deploy"

File: src/docs/guides/fixing-common-errors.md [hunk 3]
```diff
@@ -121,3 +121,15 @@ func main() {
   // ...
 }
 ```
+
+## Post requests turn into GET requests
```
Comment (melissa-hale) [inline src/docs/guides/fixing-common-errors.md:125]: i would make this POST instead of Post

Comment (brody192) [reply src/docs/guides/fixing-common-errors.md:125]: Ha! I don't know how I missed that.

File: src/docs/quick-start.md [hunk 4]
```diff
@@ -2,196 +2,182 @@
...
+
+    The CLI will now scan our project files, compress them, and upload them to Railway's backend for deployment.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1723752558/docs/quick-start/railway_up_vns3u4.png"
+alt="screenshot of the command line after railway up was run"
+layout="responsive"
+width={836} height={214} quality={100} />
 
-### Install and Link the CLI
+You can now open the link that was printed when running `railway init`
```
Comment (melissa-hale) [inline src/docs/quick-start.md:91]: Maybe mention that you can also run `railway open` ?

**Review by melissa-hale — APPROVED** [review]
Just added a tiny suggestion, unnecessary to add, will let you decide.  Otherwise - send it!

**Pull request merged**